data_pipeline/README.md

```diff
@@ -1,24 +1,23 @@
 # Traffic4cast 2022 Data Pipeline
 
-We will publish the code to generate the speed classifications from the traffic movies later in this sub-folder.
+The Traffic4cast 2022 competition dataset has inspired the new dataset and pipeline called MeTS-10 (Metropolitan Segment Traffic Speeds from Massive Floating Car Data in 10 Cities). This sub-folder contains instructions on how to use https://github.com/iarai/MeTS-10 for preparing data compatible with Traffic4cast 2022.
 
-### Color Scheme
-<img src="../img/data_pipeline_color_scheme.svg">
 
-### Symbols Used
-<img src="../img/data_pipeline_symbols.svg">
+## Static Road Graph Data
 
+The road graph data (nodes and edges) can be prepared using the scripts from from https://github.com/iarai/MeTS-10/tree/master/data_pipeline
+* `dp01_movie_aggregation.py`
+* `dp02_speed_clusters.py`
+* `dp03_road_graph.py`
 
-## Data Pipeline Node Data: Vehicle Counts
-<img src="../img/data_pipeline_node_data.svg">
+Run `dp03_road_graph.py`  with the following custom arguments:
+* `--custom_filter '["highway"~"motorway|motorway_link|trunk|primary|secondary|tertiary|unclassified|residential"]'`
+* `--parse_maxspeed` (needs to be enabled for Madrid)
+* `--heatmap_filter`
 
+The output files `road_graph_nodes.parquet`, `road_graph_edges.parquet`, `road_graph.gpkg` and `road_graph.graphml` will be written to `road_graph/<city>`.
```

Comment:
Do we want to refer to `README_DATA_SPECIFICATION.md` in both repos for the minor differences in the attributes?